### PR TITLE
Add possibility to change the role color at TargetID + Corpse Icon again

### PR DESCRIPTION
**Pull request by lebroomer**
See https://github.com/Alf21/ttt2-role_siki/pull/3

### CODE REVIEW

**Comment by Alf21** [comment]
Instead of adding new hooks, i suggest to just modify the var that is returned by ply:GetColor()

File: gamemodes/terrortown/gamemode/cl_targetid_main.lua [hunk 1]
```diff
@@ -363,6 +363,7 @@ function GM:HUDDrawTargetID()
 
 		text = L["target_" .. rd.name]
 		c = IsValid(ent) and ent:GetRoleColor() or rd.color
+		c = hook.Run("TTT2ModifyTargetIDColor", ent) or c
```
Comment (saibotk) [inline gamemodes/terrortown/gamemode/cl_targetid_main.lua]: no new line needed, just append this with an or

**Comment by Alf21** [comment]
Could you use a hook that fits the sense? eg instead of "TTT2ModifyBodyFoundRoleIcon" -> "TTT2ModifyBodyFoundRoleColor"

File: gamemodes/terrortown/gamemode/server/sv_corpse.lua [hunk 2]
```diff
@@ -211,6 +211,7 @@ function CORPSE.ShowSearch(ply, rag, covert, long_range)
 	-- init a heap of data we'll be sending
 	local nick = CORPSE.GetPlayerNick(rag)
 	local subrole = rag.was_role
+	local role_color = rag.role_color or GetRoleByIndex(subrole).color
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/server/sv_corpse.lua]: The second color (after "or") isn‘t needed :)

**Comment by Alf21** [comment]
thanks!

**Pull request merged**